code/DebugBarDatabaseCollector.php

```diff
@@ -34,12 +34,13 @@ public function collect()
         // Check for excessive number of queries
         $dbQueryWarningLevel = DebugBar::config()->warn_query_limit;
         if ($dbQueryWarningLevel && $data['nb_statements'] > $dbQueryWarningLevel) {
+            $helpLink = DebugBar::config()->performance_guide_link;
```

Comment:
@Firesphere in 3.x you can access class configuration directly on a Config_ForClass instance or using `get()` - [see here](https://github.com/silverstripe/silverstripe-framework/blob/3/core/Config.php#L916-L918) for where it's implemented